distmv/general.go

```diff
@@ -6,10 +6,11 @@
 package distmv
 
 var (
-	badQuantile      = "distmv: quantile not between 0 and 1"
-	badReceiver      = "distmv: input slice is not nil or the correct length"
-	badSizeMismatch  = "distmv: size mismatch"
-	badZeroDimension = "distmv: zero dimensional input"
+	badQuantile        = "distmv: quantile not between 0 and 1"
+	badReceiver        = "distmv: input slice is not nil or the correct length"
+	badSizeMismatch    = "distmv: size mismatch"
+	badZeroDimension   = "distmv: zero dimensional input"
+	badNonPosDimension = "distmv: non-positive dimension input"
```

Comment:
Sorry, I thought that we were using `bad` as a generic panic prefix. I guess I misunderstood the previous suggestion.